prow/getting_started.md

```diff
@@ -77,6 +77,14 @@ Go to that address in a web browser and verify that the "echo-test" job has a
 green check-mark next to it. At this point you have a prow cluster that is ready
 to start receiving GitHub events!
 
+As of 1.8 Kubernetes uses [Role-Based Access Control (“RBAC”)](https://kubernetes.io/docs/admin/authorization/rbac/) 
+to drive authorization decisions, allowing admins to dynamically configure policies. 
+To view the deck you might need to grant a user admin role in all namespaces for the cluster.
+
+```sh
+kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user="${USER}"
```

Comment:
If multiple users do this command I think they will clobber each others bindings or fail because a binding with that name already exists.
To give each binding a unique name:
s/cluster-admin-binding/cluster-admin-binding-${USER}/